readme.md

```diff
@@ -283,6 +283,64 @@ cat /data/new_zealand.polylines | docker run -i -v /data:/data pelias/interpolat
 
 the build scripts are configurable via environment variables, you will need to download your data before running the build command.
 
+To filter the TIGER data download you can set STATE_CODE environment variable to the 2 digit code of the state to be downloaded.
+The state code can found by referencing the table below. If no STATE_CODE value is found, all US data will be downloaded.
+ 
+| code | state |
+| --- | --- |
+| 01 | Alabama              |
+| 02 | Alaska               |
+| 04 | Arizona              |
+| 05 | Arkansas             |
+| 06 | California           |
+| 08 | Colorado             |
+| 09 | Connecticut          |
+| 10 | Delaware             |
+| 11 | District of Columbia |
+| 12 | Florida              |
+| 13 | Georgia              |
+| 15 | Hawaii               |
+| 16 | Idaho                |
+| 17 | Illinois             |
+| 18 | Indiana              |
+| 19 | Iowa                 |
+| 20 | Kansas               |
+| 21 | Kentucky             |
+| 22 | Louisiana            |
+| 23 | Maine                |
+| 24 | Maryland             |
+| 25 | Massachusetts        |
+| 26 | Michigan             |
+| 27 | Minnesota            |
+| 28 | Mississippi          |
+| 29 | Missouri             |
+| 30 | Montana              |
+| 31 | Nebraska             |
+| 32 | Nevada               |
+| 33 | New Hampshire        |
+| 34 | New Jersey           |
+| 35 | New Mexico           |
+| 36 | New York             |
+| 37 | North Carolina       |
+| 38 | North Dakota         |
+| 39 | Ohio                 |
+| 40 | Oklahoma             |
+| 41 | Oregon               |
+| 42 | Pennsylvania         |
+| 72 | Puerto Rico          |
+| 44 | Rhode Island         |
+| 45 | South Carolina       |
+| 46 | South Dakota         |
+| 47 | Tennessee            |
+| 48 | Texas                |
+| 49 | Utah                 |
+| 50 | Vermont              |
+| 51 | Virginia             |
+| 53 | Washington           |
+| 54 | West Virginia        |
+| 55 | Wisconsin            |
+| 56 | Wyoming              |
```

Comment:
If available, the dependencies should be included, they're quite small, unless there's a bigger reason (or simply not in the data).